### ISSUE TITLE
Decreasing mAP in validation set

### ISSUE THREAD

**Issue by uran-lajci** [ultralytics/ultralytics#18569]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Do these results show that the model is overfitting since the mAP is decreasing after around 50 epochs, especially for the bbox

![image](https://github.com/user-attachments/assets/9a008695-6422-4e83-826e-e476ae98d58a)

**Comment by Y-T-G** [comment]
Yes
